src/stores/marker.store.spec.ts

```diff
@@ -4,7 +4,7 @@ xdescribe('MarkerStore', () => {
   let markerStore: MarkerStore
 
   beforeEach(() => {
-    markerStore = new MarkerStore()
+    markerStore = MarkerStore.instance
```

Comment:
As its now an singleton this should be not needed anylonger. Just asign it once in beforeAll

Reply:
Good point.